rozy/src/main.rs

```diff
@@ -134,7 +131,15 @@ fn makefile_config_internal(makefile_var: &String, app_names: &[String]) -> Resu
     Ok(format!("{}:={}", &makefile_var, ozy_bin_dir.display()))
 }
 
-fn makefile_config(makefile_var: &String, app_names: &[String]) -> Result<()> {
+fn makefile_config(
+    makefile_var: &String,
+    app_names: &[String],
+    did_path_contain_ozy: bool,
+) -> Result<()> {
+    if !did_path_contain_ozy {
+        return Err(anyhow!("In order to create a Makefile config, please ensure that the Ozy bin dir is in the PATH"));
```

Comment:
wasn't the point of having this in _internal so that it would get output inside the `$(error` bit?

Reply:
Reading the help text for that subcommand, yes, it seems so